### PR TITLE
HRIS-126 [Markup] Create Employee Management Page

### PR DESCRIPTION
**Pull request by JohnRoyCabezas**
## Issue Link

[Backlog Link](https://framgiaph.backlog.com/board/HRIS?selectedIssueKey=HRIS-126&category=342155&assignee=405757)

## Definition of Done

- [x] HR Admins can view the list of employees and manage their information.

## Pre-condition

cd client
npm run dev

or 

run docker

## Expected Output

It should allow HR Admins to view the list of employees and manage their information.

## Screenshots/Recordings

Desktop View
image

Mobile View
image

Mobile View 2
image

### CODE REVIEW

File: client/src/pages/employee-management.tsx [hunk 1]
```diff
@@ -1,13 +1,71 @@
-import React from 'react'
 import { NextPage } from 'next'
+import classNames from 'classnames'
+import React, { useState } from 'react'
 
+import { dummyEmployeeManagement } from '~/utils/constants/dummyEmployeeManagement'
```
Comment (jsvelte) [inline client/src/pages/employee-management.tsx:5]: Make it proper importation

File: client/src/pages/employee-management.tsx [hunk 2]
```diff
@@ -1,13 +1,71 @@
...
+            'border-b border-slate-200 bg-slate-100 px-4 py-2'
+          )}
+        >
+          <GlobalSearchFilter
+            value={globalFilter ?? ''}
+            onChange={(value) => setGlobalFilter(String(value))}
+            placeholder="Search"
+          />
+          <div className="flex items-center space-x-2 text-slate-500">
+            <Button
```
Comment (jsvelte) [inline client/src/pages/employee-management.tsx:46]: Copy the `My Leaves page button` and make it a standard button for add action

**Review by jsvelte — APPROVED** [review]
# Approved 💯

File: client/src/components/molecules/EmployeeManagementTable/columns.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+import { createColumnHelper } from '@tanstack/react-table'
+
+import SortIcon from '~/utils/icons/SortIcon'
+import Avatar from '~/components/atoms/Avatar'
+import { IEmployeeManagement } from '~/utils/interfaces'
+import Tippy from '@tippyjs/react'
+import Button from '~/components/atoms/Buttons/Button'
+import { Edit, Trash2 } from 'react-feather'
+import classNames from 'classnames'
+
```
Comment (abduljalilpalala) [inline client/src/components/molecules/EmployeeManagementTable/columns.tsx:11]:
```suggestion
import React from 'react'
import Tippy from '@tippyjs/react'
import classNames from 'classnames'
import { Edit, Trash2 } from 'react-feather'
import { createColumnHelper } from '@tanstack/react-table'

import SortIcon from '~/utils/icons/SortIcon'
import Avatar from '~/components/atoms/Avatar'
import { IEmployeeManagement } from '~/utils/interfaces'
import Button from '~/components/atoms/Buttons/Button'
```

File: client/src/components/molecules/EmployeeManagementTable/MobileDisclose.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+                            rounded="full"
+                          />
+                          <div className="flex flex-col items-start">
+                            <h1 className="font-semibold">{row.original.name}</h1>
+                            <small className="text-slate-500">Web Developer</small>
+                          </div>
+                        </div>
+                      </div>
+                      <ChevronRight
+                        className={classNames('h-4 w-4 text-slate-600', open ? 'rotate-90' : '')}
```
Comment (jsvelte) [inline client/src/components/molecules/EmployeeManagementTable/MobileDisclose.tsx:58]: add this 

`className={classNames('h-4 w-4 text-slate-600 duration-300', open ? 'rotate-90' : '')}`

**Review by abduljalilpalala — CHANGES_REQUESTED** [review]
Please check comments 💬

File: client/src/components/molecules/EmployeeManagementTable/MobileDisclose.tsx [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+  return (
+    <>
+      {table.getPageCount() === 0 ? (
+        <div className="h-[50vh]">
+          <DiscloseMessage message="No Available Data" />
+        </div>
+      ) : (
+        <>
+          {table.getRowModel().rows.map((row) => (
+            <Disclosure key={row.id}>
```
Comment (jsvelte) [inline client/src/components/molecules/EmployeeManagementTable/MobileDisclose.tsx:28]: Please add Disclose Transition in the componet/templates please reuse it. Thank you.

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✅

**Pull request merged**